dotCMS/src/main/java/com/dotcms/contenttype/business/FieldFactoryImpl.java

```diff
@@ -10,13 +10,16 @@
 import com.dotcms.contenttype.transform.field.DbFieldTransformer;
 import com.dotcms.contenttype.transform.field.DbFieldVariableTransformer;
 import com.dotcms.repackage.org.apache.commons.lang.time.DateUtils;
+import com.dotcms.uuid.shorty.ShortyIdAPIImpl;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.dotcms.uuid.shorty.ShortyIdAPIImpl'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)